<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главный экран</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>138</width>
         <height>108</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">text-transform: uppercase;
  font-weight: bold;
  color: #3498db;
  text-decoration: none;
  inline: inline-block;
  padding: 10px 28px;
   -webkit-border-radius: 5px;
  -moz-border-radius: 5px;
  box-shadow: 0px 5px 20px 0px rgba(73, 139, 250, 0.57);</string>
             </property>
             <property name="text">
              <string>Логин</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">text-transform: uppercase;
  font-weight: bold;
  color: #3498db;
  text-decoration: none;
  inline: inline-block;
  padding: 10px 28px;
   -webkit-border-radius: 5px;
  -moz-border-radius: 5px;
  box-shadow: 0px 5px 20px 0px rgba(73, 139, 250, 0.57);</string>
             </property>
             <property name="text">
              <string>Пароль</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="name">
             <property name="styleSheet">
              <string notr="true">text-transform: uppercase;
  font-weight: bold;
  color: #3498db;
  border: 2px solid #3498db;
  text-decoration: none;
  inline: inline-block;
  padding: 10px 28px;
  -webkit-border-radius: 5px;
  -moz-border-radius: 5px;
  border-radius: 5px;
  box-shadow: 0px 5px 20px 0px rgba(73, 139, 250, 0.57);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="password">
             <property name="styleSheet">
              <string notr="true">text-transform: uppercase;
  font-weight: bold;
  color: #3498db;
  border: 2px solid #3498db;
  text-decoration: none;
  inline: inline-block;
  padding: 10px 28px;
  -webkit-border-radius: 5px;
  -moz-border-radius: 5px;
  border-radius: 5px;
  box-shadow: 0px 5px 20px 0px rgba(73, 139, 250, 0.57);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <bold>true</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">text-transform: uppercase;
  font-weight: bold;
  color: #3498db;
  border: 2px solid #3498db;
  text-decoration: none;
  inline: inline-block;
  padding: 10px 28px;
   -webkit-border-radius: 5px;
  -moz-border-radius: 5px;
  border-radius: 5px;
  box-shadow: 0px 5px 20px 0px rgba(73, 139, 250, 0.57);</string>
         </property>
         <property name="text">
          <string>Войти</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="styleSheet">
          <string notr="true">text-transform: uppercase;
  font-weight: bold;
  color: #3498db;
  text-decoration: none;
  inline: inline-block;
  padding: 10px 28px;
  -webkit-border-radius: 5px;
 margin-left: 65px;
  -moz-border-radius: 5px;
  box-shadow: 0px 5px 20px 0px rgba(73, 139, 250, 0.57);</string>
         </property>
         <property name="text">
          <string>Запомнить логин</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>138</width>
         <height>108</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
